src/display/text_layer.js

```diff
@@ -469,6 +462,15 @@ class TextLayer {
     return canvasContext;
   }
 
+  static #ensureCtxFont(ctx, size, family) {
+    const cached = this.#canvasCtxFonts.get(ctx);
+    if (cached && size === cached.size && family === cached.family) {
```

Comment:
nit: maybe we could add a cached structure `{ size: 0, family: ""}` when creating the context and then we can avoid the first check.

Reply:
Sure; getting rid of one check here obviously cannot hurt.